spec/features/manufacturer_spec.rb

```diff
@@ -18,4 +18,13 @@
 
     expect(page).to have_content("gouda")
   end
+
+  it "makes sure the list of manufacturers is displayed right" do
+    visit "/materials"
+    click_on "Manufacturer"
+    click_on "big ost"
```

Comment:
What does this do?

Reply:
It checks that the manufacturer is in the list of manufacturers on the search page.

Reply:
It would be good to do that more explicitly by something like `expect(page).to have_select(dropdown, :selected => selected_text)`

Also I would add that to the above test rather than a new test as that makes sense as part of the linking across from the manufacturer listing.